atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepBatchConfig.java

```diff
@@ -44,4 +44,13 @@ default void check() {
         Preconditions.checkState(candidateBatchSize() > 0, "Candidate batch size must be greater than zero");
         Preconditions.checkState(deleteBatchSize() > 0, "Delete batch size must be greater than zero");
     }
+
+    @Value.Derived
```

Comment:
nit: Is this annotation needed? `static` should already exclude it from the immutables code-gen as far as I know